app/views/propertyrepresentation/revokeAgentSummary.scala.html

```diff
@@ -18,28 +18,23 @@
 @import binders.pagination._
 @import binders.propertylinks._
 @import config.ApplicationConfig
+@this(mainLayout: views.html.mainLayout)
```

Comment:
Can we get a new line between lines 20 & 21, and 21 & 22, please?

Reply:
Done 👍🏻